gpbft/gpbft.go

```diff
@@ -957,10 +963,13 @@ func (i *instance) rebroadcast() error {
 		mbs := i.broadcasted.messagesByRound[round]
 		for _, mb := range mbs {
 			if err := i.participant.host.RequestBroadcast(mb); err != nil {
-				return err
+				// Silently log the error and proceed. This is consistent with the behaviour of
+				// instance for regular broadcasts.
+				i.log("failed to request rebroadcast %s at round %d: %v", mb.Payload.Step, mb.Payload.Round, err)
```

Comment:
Hm... it would be nice to sleep and/or backoff a little. But I guess that's a bit difficult.

Reply:
https://github.com/filecoin-project/go-f3/issues/632

Reply:
We might be able to use a simple circuit breaker at higher level (e.g. `f3`) to cover this.  Updated the issue description.